src/js/metadata.js

```diff
@@ -85,7 +85,7 @@ export default {
   'ClearOption': ['paintbrush'],
   'Cli': ['dos', 'line', 'command', 'console'],
   'Clipboard': ['document', 'list', 'report'],
-  'Clock': ['date', 'timer', 'hands', 'hour', 'watch'],
+  'Clock': ['clock', 'core', 'date', 'hands', 'hour', 'time', 'timer', 'watch'],
```

Comment:
```suggestion
  'Clock': ['core', 'date', 'hands', 'hour', 'time', 'timer', 'watch'],
```
The name of the icon is used as default search option, so it doesn't need to be added to the metadata.